docs/website/getting-started.md

```
@@ -183,103 +262,138 @@ It returns a structure similar to the following, which contains the update paylo
+    }
+  }
+"
 ```
 
 ## DefraDB Query Language (DQL)
 
 DQL is compatible with GraphQL but features various extensions.
 
-Read the [Query specification](./references/query-specification/query-language-overview.md) to discover filtering, ordering, limiting, relationships, variables, aggregate functions, and other useful features.
-
+Read its documentation [here](./references/query-specification/query-language-overview.md) to discover its filtering, ordering, limiting, relationships, variables, aggregate functions, and other useful features.
```

markdownlint-cli2 (0.22.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)